10up/ElasticPress
# Try: Co-Author Plus integration

### Description of the Change

Support querying post by guest author added by Co-Author Plus.

Fixes: #1792.

### Verification Process

1. Install and activate Co-Author Plus.
2. Activate Protected content feature.
3. Add a guest author named `mike` and assign it to some posts.
4. Go to /wp-admin/edit.php?author_name=mike and see assigned posts.



### Checklist:






### Applicable Issues

#1792 



### Changelog Entry
